cfgov/v1/management/commands/archive_events.py

```diff
@@ -0,0 +1,62 @@
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from v1.models.browse_filterable_page import (
+    BrowseFilterablePage, EventArchivePage
+)
+from v1.models.learn_page import EventPage
+
+
+class Command(BaseCommand):
+    help = 'Move event pages whose end date has passed into the archive'
+
+    def handle(self, *args, **options):
+        self.stdout.write('Searching for events to archive…')
```

Comment:
Is unicode ellipsis a preference? It was [explicitly removed](https://github.com/cfpb/consumerfinance.gov/commit/7e0721d) previously.

Reply:
Yeah, personal preference. I see that the PR for the linked commit notes that it shouldn't be a problem now that we're on Python 3, though sticking with the dots may be worth doing to be extra cautious. @chosak Would you still take that position?

Reply:
Unicode away! In Python 2 I believe we needed lines [like this](https://github.com/cfpb/consumerfinance.gov/blob/146a9af377e85617210cf2ba57d7bfac1a317361/cfgov/paying_for_college/apps.py#L1) but in Python 3 [the default encoding is already UTF-8](https://docs.python.org/3/howto/unicode.html#the-string-type).